gems/pending/util/miq-process.rb

```diff
@@ -148,100 +148,6 @@ def self.is_worker?(pid)
     command_line.include?(MiqWorker::PROCESS_TITLE_PREFIX)
   end
 
-  LINUX_STATES = {
```

Comment:
cc @bdunne @djberg96 @jrafanie Some stuff we might want to move into LinuxAdmin or potentially one of @djberg96's Sys:: gems.